app/javascript/components/Welcome.tsx

```diff
@@ -217,7 +217,7 @@ const Welcome: React.FC<WelcomeProps> = ({
               xl="5"
               className="justify-content-center content-container"
             >
-              <p>Every address can claim a fixed amount of 300,000 CKB in a month. The claimable amount will monthly update on the first day when you claim on this site.</p>
+              <p>Every address can claim a maximum of 300,000 CKB per calendar month.</p>
```

Comment:
The frontend has been separated in https://github.com/Magickbase/ckb-testnet-faucet-frontend and the corresponding code can be found at https://github.com/Magickbase/ckb-testnet-faucet-frontend/blob/4db87c004e46f5b5cc1c006fc1ca8ed171910eb6/src/sections/Content/index.tsx#L19